docs/articles/documentation/guides/advanced-guides/speed-up-test-execution.md

```diff
@@ -103,3 +104,37 @@ This approach ensures that the page model object is created only once per test r
 > Important! If you export the page model class and create an instance in every test file, JavaScript creates a new page object per test file. This increases test execution time and memory consumption.
 
 For more on page models, read [Page Model](../../guides/concepts/page-model.md).
+
+## Enable Server-Side Caching
+
+> Important! Server-side caching is **experimental**.
+>
+> Disable caching if you run into compatibility issues with your tests.
+
+TestCafe can save webpage assets (like styles, scripts, and images) in its server cache. When TestCafe visits the page again, it pulls assets from the cache instead of requesting them from server.
+
+You can enable caching in one of the following ways:
+
+* [--cache](../../reference/command-line-interface.md#cache) CLI option
+
+  ```sh
+  testcafe chrome my-tests/ --cache
+  ```
+
+* [cache]() configuration file property
+
+  ```json
+  {
+  "cache": true
+  }
+  ```
+
+* [cache]() API option
+
+  ```js
+  runner.run({ cache: true });
+  ```
+
+Enable server-side caching when you test asset-heavy webpages to decrease test run time.
```

Comment:
`you test asset-heavy webpages to decrease test run time` 
It's difficult to read. Let's try to describe in more detail.
The `cache` feature can useful if your web application loads a lot assets (scripts, styles and etc.) and browser should cache these assets.

Reply:
> If the tested application loads assets that are cached by the browser, enable server-side caching to decrease test run time.

Reply:
Let's emphasize that there are many assets:

```js
If the tested application loads A LOT OF assets...
```

Reply:
`a lot` is not very good for docs because it leaves the user wondering "how much is that exactly?" On the other hand, we can't give a specific number here because it would be arbitrary.

`asset-heavy webpages` means `applications that load a lot of assets`

I propose the following:

> To decrease test run time, enable `--cache` when you test asset-heavy web applications.